repository_rules.bzl

```diff
@@ -151,6 +151,24 @@ def switched_rules_by_language(
         "@com_google_api_codegen//rules_gapic/java:java_gapic_pkg.bzl",
     )
 
+    # Java microgenerator.
+    # These will eventually replace the monolith Java rules.
+    rules["java_microgenerator_gapic_library"] = _switch(
```

Comment:
We have been using the `2` suffix to distinguish between microgenerator and monolith while both are present in the imports (see [python_gapic_library2](https://github.com/googleapis/googleapis/blob/master/repository_rules.bzl#L173)) The idea was that microgenerator is an implementation detail, which is not important for the users of this repository. The only thing which matters is that it is some sort of a new version of the same rule, that is why it is called as `java_gapic_library2`. 

Please consider using the same naming convention here.